action/mount.go

```diff
@@ -31,7 +33,11 @@ func (s *Action) MountsPrint(c *cli.Context) error {
 		return nil
 	}
 	root := simple.New(color.GreenString(fmt.Sprintf("gopass (%s)", s.Store.Path())))
-	for alias, path := range s.Store.Mounts() {
+	mounts := s.Store.Mounts()
+	mps := s.Store.MountPoints()
+	sort.Sort(store.ByPathLen(mps))
+	for _, alias := range mps {
+		path := mounts[alias]
```

Comment:
Unlikely, possible, but it will be caught by AddMount: https://github.com/justwatchcom/gopass/blob/master/tree/simple/folder.go#L200